### DISCUSSION TITLE
Comment under the Table

### DISCUSSION THREAD

**Discussion by Anmol-Baranwal** [community/community#34092] [Other Feature Feedback, Questions, & Ideas]
## Table Formation

```
| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |
```
OUTPUT:
| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |

## Where I want to add code
- I want to add a comment or a line of code that doesn't affect the structure of the table just after every row like below:
```
| First Header  | Second Header |
| ------------- | ------------- |
<!-- -----------------THE COMMENT NEEDS TO BE PLACED HERE--------------------------------------------------------------- -->
| Content Cell  | Content Cell  |
```
OUTPUT:
| First Header  | Second Header |
| ------------- | ------------- |

| Content Cell  | Content Cell  |

## Purpose

`I need to add a comment for the ease of finding a particular row & proper separation of different rows`

Is there a way to do this? I just cannot find any answer related to this query.

**Comment by SilasRodrigues19** [comment]
@Anmol-Baranwal Try putting a | to separate the html comment and it needs to be in front of the table row

```
| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |<!--|  comment |-->
| Content Cell  | Content Cell  |
```

Result:

| First Header  | Second Header |
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |

**Reply by SilasRodrigues19** [reply]
I think u would like the comment to be below, but I hope this helps

**Reply by Anmol-Baranwal** [reply]
There is a problem with your approach -
- This could be an `alternative` but It doesn't fulfill the purpose of separating rows as it is appended to the row, suppose if the statement is 4 lines long then it will be very difficult to differentiate

> `| <a href=""><img width='700' src='https://user-images.githubusercontent.com/74038190/184916200-e51c4f7f-a875-4ea4-8f57-81e3bb8738f1.png' alt="" /></a> | [Glaze]()  <br><br> ✔ Free <br><br>Format <br> • PNG<br><br> ✌️Author   <br> • PNG |<--|  comment |-->`
`| <a href=""><img width='700' src='https://user-images.githubusercontent.com/74038190/184916200-e51c4f7f-a875-4ea4-8f57-81e3bb8738f1.png' alt="" /></a> | [Glaze]()  <br><br> ✔ Free <br><br>Format <br> • PNG<br><br> ✌️Author |<--|  comment |-->`
`| <a href=""><img width='700' src='https://user-images.githubusercontent.com/74038190/184916200-e51c4f7f-a875-4ea4-8f57-81e3bb8738f1.png' alt="" /></a> | [Glaze]()  <br><br> ✔ Free <br><br>Format <br> • PNG<br><br> ✌️Author |<--|  comment |-->`

I am writing comments in an incorrect manner so that you can see how difficult it is who see this markdown.

OUTPUT
| First Header  | Second Header |
| ------------- | ------------- |
`| <a href=""><img width='700' src='https://user-images.githubusercontent.com/74038190/184916200-e51c4f7f-a875-4ea4-8f57-81e3bb8738f1.png' alt="" /></a> | [Glaze]()  <br><br> ✔ Free <br><br>Format <br> • PNG<br><br> ✌️Author   <br> • PNG |`
`| <a href=""><img width='700' src='https://user-images.githubusercontent.com/74038190/184916200-e51c4f7f-a875-4ea4-8f57-81e3bb8738f1.png' alt="" /></a> | [Glaze]()  <br><br> ✔ Free <br><br>Format <br> • PNG<br><br> ✌️Author |`
`| <a href=""><img width='700' src='https://user-images.githubusercontent.com/74038190/184916200-e51c4f7f-a875-4ea4-8f57-81e3bb8738f1.png' alt="" /></a> | [Glaze]()  <br><br> ✔ Free <br><br>Format <br> • PNG<br><br> ✌️Author |`

`How well do you think someone can differentiate between rows and read the comment if there are at least 50 rows?`

I hope you understand !! Your approach might help in certain scenarios though.
